src/latbin/lattice-lmrescore-const-arpa-kaldi-rnnlm-pruned.cc

```diff
@@ -0,0 +1,181 @@
+// latbin/lattice-lmrescore-kaldi-rnnlm-pruned.cc
+
+// Copyright 2017 Johns Hopkins University (author: Daniel Povey)
```

Comment:
I don't believe it's necessary to have an extra program for the const-arpa case.  You could just have an option to lattice-lmrescore-kaldi-rnnlm-pruned.  They OnDemandDeterministicFst objects share the same interface, so the if-statement wouldn't have to encompass the bulk of the code, just the code that initializes that object.

Reply:
Oh sure. I thought about having both in one binary but then I saw there are separate "lattice-lmrescore" and "lattice-lmrescore-const-arpa" under latbin/ so I separated them.

I will make the changes.